runtime/src/main/kotlin/kotlin/test/DefaultAsserter.kt

```diff
@@ -8,7 +8,7 @@ package kotlin.test
 /**
  * Default [Asserter] implementation for Kotlin/Native.
  */
-public object DefaultAsserter : Asserter {
+public object DefaultObjectAsserter : Asserter {
```

Comment:
Perhaps we can have an instance of common `DefaultAsserter` in a `@SharedImmutable` top-level val instead?

Reply:
There are no need actually to have `DefaultAsserter` under this annotation. Maybe it could be better to fix todo here and use this asserter https://github.com/JetBrains/kotlin/blob/2e12834b6a529f63e7a7955f4b5c4db24ee7adf9/libraries/kotlin.test/common/src/main/kotlin/kotlin/test/DefaultAsserter.kt#L12